pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -644,7 +550,8 @@ def handle_command_generate_args(  # noqa: C901
 
     # set linker and C flags to error on anything to do with function declarations being wrong.
     # Better to fail at compile or link time.
-    if is_link_command:
+    is_link_cmd = get_library_output(line) is not None
+    if is_link_cmd:
```

Comment:
Maybe replace `get_library_output` with `is_link_cmd` since it seems like we don't do anything with the return value of `get_library_output` anymore:
```suggestion
    if is_link_cmd(line):
```

Reply:
Makes sense. Thanks!